test/api-digester/source-stability.swift.expected

```diff
@@ -25,6 +25,14 @@ Var Set.startIndex has declared type change from SetIndex<Element> to Set<Elemen
 Func Dictionary.index(after:) has return type change from DictionaryIndex<Key, Value> to Dictionary<Key, Value>.Index
 Func Dictionary.index(forKey:) has return type change from DictionaryIndex<Key, Value>? to Dictionary<Key, Value>.Index?
 Func Dictionary.remove(at:) has 1st parameter type change from DictionaryIndex<Key, Value> to Dictionary<Key, Value>.Index
+
+/* False positives due to archetype -> interface type change */
```

Comment:
This file doesn't need to be sorted, so it's probably better to keep the Set/Dictionary stuff together.